python/py-conllu/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        EmilStenstrom conllu 68199a7
```

Comment:
don't specify a commit hash, get a version... It appears that upstream does not tag versions properly on GitHub. In that case get them from PyPI instead.

Reply:
Thanks done. 🦃🦃🦃